dev/archery/archery/integration/runner.py

```diff
@@ -137,8 +137,6 @@ def _gold_tests(self, gold_dir):
                 skip.add("Go")
                 skip.add("JS")
                 skip.add("Rust")
-            if name == 'zstd':
```

Comment:
A downside to removing this comment completely is that someone might not easily figure out how to skip tests based on name (when working on another implementation).

Do you mind adding a comment to the effect that if an implementer is skipping one of the compression formats, they could use `if name == 'format'`?

Reply:
Added a link back to this PR>